front/src/styles/scss/applications/operationalStudies/_scenario.scss

```
@@ -536,26 +537,118 @@
           }
         }
 
-        .filter {
+        // Base mixin for all toolbar buttons
+        @mixin toolbar-button-base($is-grey-color: true, $has-hover: true, $has-active: true) {
+          @if $is-grey-color {
+            color: var(--grey40);
+          } @else {
+            color: var(--primary60);
+          }
+
+          @if $has-hover {
+            &:hover {
+              color: var(--grey60);
+            }
+          }
+
+          @if $has-active {
+            &.active {
+              color: var(--grey90);
+            }
+          }
+
+          &:disabled {
+            color: var(--grey20);
+            pointer-events: none;
+          }
+        }
+
+        .button-toolbar {
           background-color: var(--ambientB10);
-          height: 31px;
+          min-height: 32px;
           border-bottom: solid 1px rgba(0, 0, 0, 0.25);
           display: flex;
+          align-items: center;
           justify-content: center;
+          position: relative;
+          padding-bottom: 5px;
+
+          .toolbar-button {
+            @include toolbar-button-base();
+          }
+
+          .select-options-button {
+            @extend .toolbar-button;
+            margin-left: 9px;
```

This seems to be better for the horizontal alignment with the checkbox below

```suggestion
            margin-left: 10px;
```

---

fixed